README.md

```diff
@@ -249,53 +277,142 @@ pre-commit install
 ```
 
 
-## Running DataGateway API
+
+
+# Running DataGateway API
 Depending on the backend you want to use (either `db` or `python_icat`) the connection
 URL for the backend needs to be set. These are set in `config.json` (an example file is
 provided in the base directory of this repository). Copy `config.json.example` to
 `config.json` and set the values as needed.
 
-Ideally, the API would be run with:
-`poetry run python -m datagateway_api.src.main`
-However it can be run with the flask run command as shown below:
+By default, the API will run on `http://localhost:5000` and all requests are made here
+e.g. `http://localhost:5000/sessions`.
+
 
+## API Startup
+Ideally, the API would be run using the following command:
```

Comment:
Is there a reason that it should be "ideally" done with `poetry` and not `flask`? Or is it just the point below about flask not taking the `config.json` options into account?

Reply:
I wasn't entirely sure either (this section predates my time with the API), but I found out that `flask run` should only be run for development purposes. I've reworded this section and added a link to specifically where the Flask docs advise against production use of the command. Obviously ignoring `config.json` is also a reason too.